packages/uikit/src/molecules/FormGroup/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from '../../lib/styledComponents';
+
+import { FormGroupLabel, Caption } from '../../atoms/Typography';
+import theme from '../../theme';
+
+interface Props {
+  label: string;
+  required?: boolean;
+  errorMsg?: string;
+  children: any;
+}
+
+const FormGroupWrapper = styled.div``;
+
+const FormGroup: React.SFC<Props> = ({
+  label,
+  required,
+  children,
+  errorMsg
+}) => (
+  <FormGroupWrapper>
+    <FormGroupLabel>
+      {label} {required ? '*' : null}
+    </FormGroupLabel>
+    {React.cloneElement(children, { hasError: !!errorMsg })}
```

Comment:
Can you explain this please?

Reply:
Basically saves us having to pass the `hasError` flag to the input component manually. So when it's passed to the form group it gets applied to the child automatically

Reply:
ahhh smart 👍